### PR TITLE
PeakVI user guide

### LINKED ISSUE #1172: User guide: PeakVI

### PR DESCRIPTION
**Pull request by talashuach**
closes #1172

### CODE REVIEW

File: docs/user_guide/models/peakvi.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+PeakVI posits that the observed value for cell :math:`i` and region :math:`j`, :math:`x_{ij}`, 
+is generated by the following process:
+
+.. math::
+   :nowrap:
+
+   \begin{align}
+    z_i &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
+    y_{ij} &= g_z^j\left(z_i, s_i\right) \\
+    \ell_i &= f_\ell\left(x_i\right)
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:50]:
```suggestion
    \ell_i &= f_\ell\left(x_i\right)\\
```

File: docs/user_guide/models/peakvi.rst [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`z_i \in \mathbb{R}^d`
+     - Low-dimensional representation capturing the state of a cell
+     - ``z``
+   * - :math:`\y_i \in \left[0,1\right]^{M}`
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:81]:
```suggestion
   * - :math:`y_i \in \left[0,1\right]^{M}`
```

File: docs/user_guide/models/peakvi.rst [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+
+Transfer learning
+-----------------
+
+A PeakVI model can be pre-trained on reference data and updated with query data using :func:`~scvi.model.PEAKVI.load_query_data`, which then facilitates transfer of metadata like cell type annotations. See the :doc:`/user_guide/background/transfer_learning` guide for more information.
+
+
+Estimation of accessibility
+------------------------------------------------
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:137]: I thought you feed the mean of z through the decoder? We should note this here

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: docs/user_guide/models/peakvi.rst (hunk 3 above)
Comment (talashuach) [reply docs/user_guide/models/peakvi.rst]: yup, my bad. Fixed.

File: docs/user_guide/models/peakvi.rst [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+    + Comprehensive in capabilities.
+
+    + Scalable to very large datasets (>1 million cells).
+
+The limitations of peakVI include:
+
+    + Effectively requires a GPU for fast inference.
+
+    + Latent space is not interpretable, unlike that of a linear method.
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:18]:
```suggestion
- Comprehensive in capabilities.
- Scalable to very large datasets (>1 million cells).

The limitations of peakVI include:

- Effectively requires a GPU for fast inference.
- Latent space is not interpretable, unlike that of a linear method.
```

File: docs/user_guide/models/peakvi.rst [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+PeakVI uses variational inference, specifically auto-encoding variational Bayes (see :doc:`/user_guide/background/variational_inference`) to learn both the model parameters (the neural network params, scaling factors, etc.) and an approximate posterior distribution with the following factorization:
+
+ .. math::
+    :nowrap:
+
+    \begin{align}
+       q_\eta(z_i, \ell_i \mid x_i) :=
+       q_\eta(z_i \mid x_i)q_\eta(\ell_i \mid x_i).
+    \end{align}
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:102]:
```suggestion
.. math::
   :nowrap:

   \begin{align}
      q_\eta(z_i, \ell_i \mid x_i) :=
      q_\eta(z_i \mid x_i)q_\eta(\ell_i \mid x_i).
   \end{align}
```

File: docs/user_guide/models/peakvi.rst [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+In :func:`~scvi.model.PEAKVI.get_accessibility_estimates` PeakVI returns the expected value of :math:`y_i` under the approximate posterior. For one cell :math:`i`, this can be written as:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+       \mathbb{E}_{q_\eta(z_i \mid x_i)}\left[g_z\left( z_i, s_i \right) \right],
+    \end{align}
+
+The expectation is computed using the mean of :math:`z_i` by default, but this behaviour can be changed by setting ``use_z_mean=False`` argument.
```
Comment (adamgayoso) [inline docs/user_guide/models/peakvi.rst:148]:
```suggestion
As the expectation can be expensive to compute, by default, PeakVI uses the mean of :math:`z_i` as a point estimate, but this behaviour can be changed by setting ``use_z_mean=False`` argument.
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

**Pull request merged**